### DISCUSSION TITLE
Can MAUI support the AppWidget in iOS or Android ?

### DISCUSSION THREAD

**Discussion by JeffCaptain** [dotnet/maui#9456] [Q&A] [answered]
like title

**Comment by xtuzy** [comment, marked as answer]
I get some information from google.

In Android, you can do it all use c#.
- https://marcofolio.net/app-widget-xamarin-android/

In iOS, you can use swift write widget, add this widget to your maui app.
- https://michaeldimoudis.com/ios-14-widgets-with-xamarin-forms-e186d24a0638
- https://www.youtube.com/watch?v=CJuLYvnxQTE

**Reply by JeffCaptain** [reply]
Woo nice, thank you very much .  No wonder I can't find the relevant information ,  I ignored the keyword "Xamarin Forms".

**Reply by LuohuaRain** [reply]
Is there any way only using .NET MAUI or Razor components to implement AppWidget?

**Reply by xtuzy** [reply]
i don't know, you need google the detail. on android, maui is android app, if android app can do it, should can be done. on ios, some thing is not ios app, maui only support develop ios app, such as extension, ios watch app is not be support.
